README.md

```diff
@@ -117,4 +117,4 @@ Feedstock Maintainers
 =====================
 
 * [@lheagy](https://github.com/lheagy/)
-
+* [@jcapriot](https://github.com/jcapriot)
```

Comment:
The README is auto-generated all you need to change is the recipe and then re-render.